lib/hasher/src/SHA1Strategy.cpp

```diff
@@ -90,7 +90,11 @@ namespace irods {
         unsigned long out_len = CHKSUM_LEN - len;
 
         unsigned char out_buffer[CHKSUM_LEN];
-        base64_encode( final_buffer, SHA_DIGEST_LENGTH, out_buffer, &out_len );
+        int rc = base64_encode(final_buffer, SHA_DIGEST_LENGTH, out_buffer, &out_len);
+        if (0 != rc) {
+            const auto msg = fmt::format("{}: Failed to base64 encode hash.", __func__);
+            return ERROR(rc, msg);
```

Comment:
```suggestion
            auto msg = fmt::format("{}: Failed to base64 encode hash.", __func__);
            return ERROR(rc, std::move(msg));
```

or ...


```suggestion
            return ERROR(rc, fmt::format("{}: Failed to base64 encode hash.", __func__));
```

_This applies to the other hashers too._

Reply:
Done. Since it was trivial, I went ahead and did the same for the other error messages in the hashers.